web/src/utils/formatting.ts

```diff
@@ -10,7 +10,7 @@ export const formatPower = function (d: number, numberDigits = DEFAULT_NUM_DIGIT
     return d;
   }
   const power = `${d3.format(`.${numberDigits}s`)(d * 1e6)}W` //Add a space between the number and the unit
-    .replace(/([A-Z])/, ' $1')
+    .replace(/([A-Z, a-z])/, ' $1')
```

Comment:
Nice! But you can combine these like this:
```suggestion
    .replace(/([a-zA-Z])/, ' $1')
```